src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -71,6 +71,33 @@ AttributRadiationFlag
 
 /*########################### define species #################################*/
 
+/*--------------------------- photons -------------------------------------------*/
+
+#if(ENABLE_SYNCHROTRON_PHOTONS == 1)
```

Comment:
Here, the precompiler statement is necessary because it's the only occurrence in the code. :)